modules/gapi/test/infer/gapi_infer_ie_test.cpp

```diff
@@ -372,9 +374,8 @@ TEST_F(ROIList, TestInfer)
     std::tie(age, gender) = cv::gapi::infer<AgeGender>(rr, in);
     cv::GComputation comp(cv::GIn(in, rr), cv::GOut(age, gender));
 
-    auto pp = cv::gapi::ie::Params<AgeGender> {
-        params.model_path, params.weights_path, params.device_id
-    }.cfgOutputLayers({ "age_conv3", "prob" });
+    auto pp = cv::gapi::ie::Params<AgeGender> {model_path, weights_path, device_id
+                                              }.cfgOutputLayers({ "age_conv3", "prob" });
```

Comment:
Something strange happened to this formatting.

it is not supposed to look like this.

Reply:
I can format to the indentation like it was before in this snippet

Reply:
@dmatveev please check if OK now